service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/ManageLegalRepresentativesValidatorTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.service.OrganisationService;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.enums.LegalRepresentativeRole.EXTERNAL_LA_BARRISTER;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ExtendWith(MockitoExtension.class)
+class ManageLegalRepresentativesValidatorTest {
```

Comment:
maybe it is worth to test scenario with multiple representatives?

Reply:
done, now it gives the item number